source/docs/articles/local/developing-locally.md

```diff
@@ -5,7 +5,8 @@ category:
 - getting-started
 keywords: local development, pantheon, develop locally, developing
 ---
-While Pantheon provides several options for on-server development, local development has a number of advantages, especially if continuous Internet access is a concern. Pantheon cannot troubleshoot or support local development solutions, however we can provide some suggestions and known working solutions.
+Local development has a number of advantages, especially if continuous Internet access is a concern. This article will walk you through setting up a local development environment.
```

Comment:
Suggest restoring original intro within this doc, given that the duplicate intro in the local.md landing page is deleted. The forewarning about Pantheon's inability to troubleshoot local developments should stay